packages/react-ui-components/src/DateInput/dateInput.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import enhanceWithClickOutside from 'react-click-outside';
 import moment from 'moment';
 import mergeClassNames from 'classnames';
+import I18n from '@neos-project/neos-ui-i18n';
```

Comment:
Please remove this dependency and pass the needed labels from the outside (`neos-ui-editors`)